mysite/ct/serializers.py

```diff
@@ -0,0 +1,104 @@
+from rest_framework import serializers
+
+from ct.models import Response, StudentError, UnitLesson
+
+
+class ResponseSerializer(serializers.ModelSerializer):
+    author_id = serializers.IntegerField()
+    author_name = serializers.SerializerMethodField()
+    lti_identity = serializers.SerializerMethodField()
+    unitLesson_id = serializers.IntegerField()
+    unit_id = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Response
+        fields = (
+            'id',
+            'author_id',
+            'author_name',
+            'lti_identity',
+            'text',
+            'confidence',
+            'selfeval',
+            'status',
+            'unitLesson_id',
+            'unit_id',
+        )
+
+    def get_author_name(self, obj):
+        """
+        Returning author name.
+        """
+        name = obj.author.get_full_name()
+        if not name:
+            name = obj.author.username
+        return name
+
+    def get_lti_identity(self, obj):
+        """
+        Returning LTI user_id.
+        """
+        lti_user = obj.author.lti_auth.first()
+        if lti_user:
+            return lti_user.user_id
+
+    def get_unit_id(self, obj):
+        """
+        Returning Unit id.
+        """
+        return obj.unitLesson.unit.id
```

Comment:
instead of making `get_unit_id` method you can use `serializers.ReadOnlyField(source='unitLesson.unit.id')`